.github/workflows/static.yml

```diff
@@ -0,0 +1,59 @@
+name: Static analysis
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  phpstan:
+    name: PHPStan
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v1
+
+      - name: Install
+        uses: docker://composer
+        with:
+          args: install --ignore-platform-reqs
+
+      - name: PHPStan
+        uses: docker://oskarstark/phpstan-ga
+        with:
+          entrypoint: ./.github/workflows/phpstan.entrypoint
+          args: analyze --no-progress
+
+  php-cs-fixer:
+    name: PHP-CS-Fixer
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v1
+
+      - name: PHP-CS-Fixer
+        uses: docker://oskarstark/php-cs-fixer-ga:2.14.0
+        with:
+          args: --dry-run --diff-format udiff
```

Comment:
:/ what are the rules that this is using? is it me or a .php_cs.dist file does not exist on the current PR?

Reply:
Rules are already present in the repository